### PR TITLE
feat(css): Add CSS Values Level 4 `abs()` compatibility data

### PR DESCRIPTION
**Pull request by ramiy**
Add browser compatibility data for the CSS abs() function to contain the new additions from CSS Values and Units Level 4.

Related: https://github.com/mdn/content/pull/5168

### CODE REVIEW

**Comment by ddbeck** [comment]
Setting as not ready because this is blocked by https://github.com/mdn/browser-compat-data/issues/10619.

**Comment by saschanaz** [comment]
That's added via https://github.com/w3c/csswg-drafts/commit/8e3fb63a705e6c4bccb432fb8ca417423119303c by @tabatkins, but interestingly not via a pull request and I don't see any tracking issues for browser implementations. Not sure what's going on there 🤔

**Comment by tabatkins** [comment]
I'm not sure what you think I'm supposed to have done here? abs() was added as part of the larger suite of new math functions, discussed and resolved on in a previous CSS face-to-face meeting.

**Comment by saschanaz** [comment]
I wanted to check interest from implementers, and couldn't do it from the commit itself. What's the best way to track such thing?

**Comment by tabatkins** [comment]
Depends on what level of "interest" you want. It was part of the suite of math functions approved at the CSSWG face-to-face, and has the same level of approval/interest as sin()/etc. Minutes are recorded at <https://lists.w3.org/Archives/Public/www-style/2020Feb/0007.html>

**Comment by ramiy** [comment]
Guys, I'm planning to create MDN pages for all the [CSS Math functions](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions#math_functions). I need to know whether to create [BCD json files](https://github.com/mdn/browser-compat-data/issues/10619) for those functions. Note that no browser support them, yet. But I'm planning to nag the browser dev teams to implement them.

**Comment by queengooborg** [comment]
Hey @ramiy, thank you for your patience on this -- while we are still deciding whether to allow or disallow all-false features in BCD, a new `spec-url` meta tag has been added to content files, which can be used for features that aren't in BCD for one reason or another.  This could be used here for these functions, if browser support is still all false!

**Comment by queengooborg** [comment]
Hey again, thank you for your patience on this! We have decided not to include features that have no compatibility in BCD. A spec may implement a feature, but then rename or remove it before any browsers implement it, thus leaving a leftover statement in BCD.

Would you be down to research if any browsers support this feature now? If not, I'd be happy to look into it and follow up!

**Comment by ramiy** [comment]
Hi @queengooborg @ddbeck @wbamberg,

Looks like WebKit implemented this function. See bug  _#229786_: _[Implement abs,sign calc functions](https://bugs.webkit.org/show_bug.cgi?id=229786)_ _(Sep 2021)_

Also, according to the _Web Platform Tests Dashboard_, the function works:
- https://wpt.fyi/results/css/css-values/signs-abs-computed.html?label=master&label=stable&aligned&view=subtest
- https://wpt.fyi/results/css/css-values?label=master&label=stable&aligned&view=subtest&q=sign

But I couldn't find any mention in official releases.

**Comment by saschanaz** [comment]
Nice find! `CSS.supports("width: abs(20% - 100px)")` reports true already on Safari 15.6.

**Comment by ramiy** [comment]
@saschanaz 

Web Platform Tests allows you to edit the search and specify the specific browser versions.

I checked all the available Safari versions. Version 15.4 is the first version to add support for `abs()` and `sign()` functions.

Ref: https://wpt.fyi/results/css/css-values?label=master&label=stable&product=chrome&product=edge&product=firefox&product=safari-15.4%20%2816613.1.17.1.11%29&aligned&view=subtest&q=sign

Before that, no support.

Ref: https://wpt.fyi/results/css/css-values?label=master&label=stable&product=chrome&product=edge&product=firefox&product=safari-15.3%20%2815612.4.9.1.8%29&aligned&view=subtest&q=sign

**Comment by ramiy** [comment]
Moving the conversation to https://github.com/mdn/browser-compat-data/issues/17868

**Comment by ramiy** [comment]
@queengooborg 

See [Safari 15.3](https://wpt.fyi/results/css/css-values?label=master&label=stable&product=chrome&product=edge&product=firefox&product=safari-15.3%20%2815612.4.9.1.8%29&aligned&view=subtest&q=sign) vs. [Safari 15.4](https://wpt.fyi/results/css/css-values?label=master&label=stable&product=chrome&product=edge&product=firefox&product=safari-15.4%20%2816613.1.17.1.11%29&aligned&view=subtest&q=sign) support in web platform tests.

**Review by queengooborg — APPROVED** [review]

**Pull request merged**
